backend/composer/models.py

```diff
@@ -386,14 +386,15 @@ class ConnectivityStatement(models.Model):
     owner = models.ForeignKey(
         User, verbose_name="Curator", on_delete=models.SET_NULL, null=True, blank=True
     )
+
     laterality = models.CharField(
-        max_length=20, default=Laterality.UNKNOWN, choices=Laterality.choices
+        max_length=20, choices=Laterality.choices, null=True
```

Comment:
@D-GopalKrishna question: did you also test the Django Admin interface? Shouldn't we add `blank=True` too?

Reply:
corrected @zsinnema. Thanks!